### DISCUSSION TITLE
Auditory 2AFC task?

### DISCUSSION THREAD

**Discussion by esi6** [jspsych/jsPsych#1070] [Q&A]
Hi,

I want to create an experiment that wasn't covered in the Jspsych tutorial over the summer. Mainly, an auditory 2AFC task, where the participant hears one word, followed by a brief pause, followed by another word. They then provide a response about which one was the target item.

I tried adapting the tutorials to my purposes, but I ran into an issue with the responses. In the tutorial, the responses (such as in the lexical decision task) appear to be locked to a visual stimulus. I tried utilizing this setup by simply replacing the visual stimulus with an auditory one, but because my audio files are so short (~2sec), the responses aren't getting logged and the experiment is unable to progress beyond the first test trial.

Is there a way to lock responses to a test slide, like in psychopy or e-prime? Or another way to get around this issue? I do not necessarily want participants responses to be locked to the second item, as I want to give them sufficient time to respond.

Thanks!

**Comment by becky-gilbert** [comment]
Hi @esi6! I would suggest capturing the response during the second audio trial, but setting `trial_duration` to null, `trial_ends_after_audio` to false, and `response_ends_trial` to true. That way the trial will wait forever for a response, so the participant will be able to take as long as they need.
